pandas/tseries/tests/test_resample.py

```diff
@@ -361,6 +361,29 @@ def test_fillna(self):
         with self.assertRaises(ValueError):
             r.fillna(0)
 
+    def test_fill_value(self):
```

Comment:
these tests should be in ``class Base``. Which will tests with all index types (you need to tests both Series/DataFrame here)

use ``self.create_index()`` to create your index.